pkgs/development/tools/click/default.nix

```
@@ -0,0 +1,90 @@
+{ lib
+, fetchFromGitLab
+, buildPythonPackage
+, autoconf
+, debian
+, automake
+, perl
+, libtool
+, vala
+, pkg-config
+, libgee
+, json-glib
+, properties-cpp
+, gobject-introspection
+, getopt
+, setuptools
+, pygobject3
+, wrapGAppsHook
+}:
+
+buildPythonPackage {
+  pname = "click";
+  version = "22-02-2023";
+
+  src = fetchFromGitLab {
+    owner = "ubports";
+    repo = "development/core/click";
+    rev = "aaf2735e8e6cbeaf2e429c70136733513a81718a";
+    sha256 = "sha256-pNu995/w3tbz15QQVdVYBnWnAoZmqWj1DN/5PZZ0iZw=";
+  };
+
+
+  configurePhase = ''
+    ./autogen.sh
+    ./configure --prefix=$out \
+        --with-systemdsystemunitdir=/lib/systemd/system \
+        --with-systemduserunitdir=/usr/lib/systemd/user
+  '';
+
+  buildPhase = ''
+    make
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+    cp -r {bin,click_package,lib} $out
+  '';
+
+  preFixup = ''
+    makeWrapperArgs+=(--prefix GI_TYPELIB_PATH : "$out/lib/click")
+  '';
+
+
+  format = "other";
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    perl
+    libtool
+    pkg-config
+    gobject-introspection.dev
```

Should work without the explicit `.dev`?

```suggestion
    gobject-introspection
```